.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+       node-version: '12.x'
+    - name: Install Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.7'
+        architecture: 'x64'
+    - name: Install dependencies
+      run: python -m pip install jupyterlab
+    - name: Build and install the extension
+      run: |
+        pip install .
+
+        # This doesn't work???
+        # jupyter labextension list | grep -ie "jupyterlab-kernelspy.*enabled.*ok" -
```

Comment:
Maybe something like in the cookiecutter would work? (or a similar trick)

https://github.com/jupyterlab/extension-cookiecutter-ts/blob/cb9ca8c37bf1c5c28be61b31e6fcd0cf60487a5f/.github/workflows/main.yml#L52-L53

Reply:
Note that this is commented out because I couldn't get grep to work correctly 😢

Reply:
ah, I misread what you wrote, I'll give that one a try.

Reply:
But yeah, I did actually find the problem on Friday but forgot about it: grep should be running against stderr in this case (where log entries go), as that code fixes 👍